mungegithub/mungers/mungerutil/util.go

```diff
@@ -16,14 +16,122 @@ limitations under the License.
 
 package mungerutil
 
-import "github.com/google/go-github/github"
+import (
+	"strings"
 
-// FilterValid filters a list of user data and returns only user data that contains valid username.
-func FilterValid(users []*github.User) (res []*github.User) {
```

Comment:
Looks like getInvolvedUsers and FilterValid were doing overlapping things before. Good that we got rid of it. :)